test/testutils.js

```diff
@@ -420,24 +420,43 @@ testUtils.assertStylesheet = function assertStylesheet(
  * Injecting content into a fixture and return queried element within fixture
  *
  * @param {String|Node} content to go into the fixture (html or DOM node)
+ * @param {String?} query - the CSS selector query to find target DOM node
```

Comment:
```suggestion
 * @param {String} [query=#target] - the CSS selector query to find target DOM node
```

Reply:
This has been added, though my VS Code install doesn't display #target